config.go

```diff
@@ -845,17 +842,17 @@ func (c *Settings) validateAndDefault() error {
 	return nil
 }
 
-// configSection defines a sub-document in the evegreen config
+// ConfigSection defines a sub-document in the evegreen config
 // any config sections must also be added to registry.go
-type configSection interface {
+type ConfigSection interface {
 	// id() returns the ID of the section to be used in the database document and struct tag
 	id() string
```

Comment:
Might as well make this public.

Reply:
I think this should be private since nothing outside of this package should care about the ID

Reply:
right, but we might define implementations of ConfigSection outside of this package, and I think that... doesn't work?

Reply:
I renamed it to SectionId (needed to avoid conflicts with the Id field name)